utils/lpac/files/lpac.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
This can not be done via init script?

Reply:
I did not want to do this in order to leave the option open to support multiple devices with stored configuration.